src/callstack.cr

```diff
@@ -152,6 +162,14 @@ struct CallStack
     end
   end
 
+  private def colorize_address(address)
```

Comment:
This method could be more generalized and used in all places in both files. It should probably be an (undocumented) class method on `Exception`.

```crystal
def Exception.colorize(code, string)
  if colorize?
    "#{code}#{string}\e[0m"
  else
    string
  end
end
```

Reply:
Added this method into `Exception` with the new commit!